doc/how-to/ceph_networking.rst

```diff
@@ -0,0 +1,160 @@
+.. _howto-ceph-networking:
+
+How to configure Ceph networking
+================================
+
+In the ``microcloud init`` setup, you are asked if you want to provide custom subnets for the Ceph cluster.
+Here are the questions you will be asked:
+
+- `What subnet (either IPv4 or IPv6 CIDR notation) would you like your Ceph public traffic on? [default: 203.0.113.0/24]: <answer>`
+- `What subnet (either IPv4 or IPv6 CIDR notation) would you like your Ceph internal traffic on? [default: 203.0.113.0/24]: <answer>`
+
+You can choose to skip both questions (just click `Enter`) and use the default value which is the subnet used for the internal MicroCloud traffic.
+This is referred as a `usual Ceph networking setup`.
+
+But sometimes, you want to be able to use different network interfaces for some Ceph related usages.
+Let's imagine you have machines with network interfaces that are tailored for high throughput and low latency data transfer,
+like 100GbE+ QSFP links, and other ones that might be more suited for ‘management’ traffic, like 1GbE or 10GbE links.
+
+In this case, it would probably be ideal to set your Ceph internal (or cluster) traffic on the high throughput network interface and the Ceph public traffic on the management network interface. This is referred as a `fully disaggregated Ceph networking setup`.
+You could also decide to put both type of traffic on the same high throughput and low latency network interface. This is referred as a `partially disaggregated Ceph networking setup`.
+
+Here is a picture of the different Ceph network setups we talked about:
+
+.. figure:: /images/ceph_networking_setups.png
+   :alt: The different Ceph networking setups available
+   :align: center
+
+Now, let's see how to configure the different Ceph networking setups in the MicroCloud initialisation process.
+
+1. First, just like when you created an uplink network for MicroCloud so that the cluster members could have external connectivity, you will need to create a dedicated network for the Ceph cluster members to communicate with each other. Let's call it ``cephbr0``::
```

Comment:
These are a lot of steps, which makes it hard to follow. Can you group the ones that belong together and use sub-steps?

As an example, you could combine steps 1-4 (I'm not writing the full text here, this is just to give you an idea what I mean):

1. Create the dedicated networks for Ceph:
    a. Create a network for internal traffic: ...
    b. For a fully disaggregated setup, create a second network for the public traffic: ...
    c. Find the assigned IP addresses:
       Maybe also for the second network:

Reply:
Would it maybe be sufficient to give the full instructions for a fully disaggregated setup only, and then add a paragraph at the end that for a partially disaggregated setup, you do this with only one network and leave either the internal or the public traffic on the default MicroCloud internal subnet?

I feel that would make the instructions way more straight-forward, and it should be easy to follow.

Reply:
fixed